src/lib/components/Header/Header.svelte

```diff
@@ -42,7 +60,7 @@
 
       <a class="ldaf-logo__compact" href="/">
         <!-- TODO: Replace alt value with content from CMS. -->
-        <img src={ldafLogo} alt="Louisiana Department of Agriculture and Forestry Home" />
+        <Logo />
```

Comment:
nit: we can remove the `TODO` comment since you moved it into the `Logo` component.